pages/windows/robocopy.md

```diff
@@ -0,0 +1,32 @@
+# robocopy
+
+> Robust File and Folder Copy.
+> By default files will only be copied if the source and destination have different time stamps or different file sizes.
+
+- Copy all .jpg and .bmp files from one directory to another:
+
+`robocopy {{path/to/source}} {{path/to/destination}} *.jpg *.bmp`
+
+- Copy all files and subdirectories, including empty ones:
+
+`robocopy {{path/to/source}} {{path/to/destination}} /E`
+
+- Mirror/Sync a directory, deleting anything not in source and include all attributes and permissions:
+
+`robocopy {{path/to/source}} {{path/to/destination}} /MIR /COPYALL`
+
+- Copy all files and subdirectories, excluding source files that are older than destination files:
+
+`robocopy {{path/to/source}} {{path/to/destination}} /E /XO`
+
+- List all files (copy dry run) 50 MBytes or larger in size:
```

Comment:
"copy dry run" seems a bit odd. How about - "List all files ... in size instead of copying them" ?